src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -629,7 +643,9 @@ void KOSMAPOSLReconstruction<TargetT>::estimate_stand_dev_for_anatomical_image(s
         const int max_z = (*anatomical_prior_sptr[i]).get_max_index();
 
         dim_z = max_z -min_z+1;
-
+        #ifdef STIR_OPENMP
+        #pragma omp parallel for reduction(+:kmean,nv)
```

Comment:
sadly not supported by Visual Studio I believe (do check). Easiest might be to check OPENMP version, and only include this pragma if it's recent enough.